src/transcription/live.rs

```diff
@@ -89,16 +137,34 @@ struct FileChunker {
 }
 
 impl Transcription<'_> {
-    pub fn stream_request<E, S: Stream<Item = std::result::Result<Bytes, E>>>(
-        &self,
-    ) -> StreamRequestBuilder<S, E> {
+    pub fn stream_request<E, S>(&self) -> StreamRequestBuilder<'_, S, E>
+    where
```

Comment:
So `StreamRequestBuilder` has the generics `<'_, S, E>`, but the `stream_request` function has the generics `<E, S>`, which means the user can't develop an intuition for which generic is which, because they are in opposite orders.  The E should come before the S in both cases, or vice versa.  (Ignore the lifetime generic.  Those always come first by definition).